localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -636,7 +656,16 @@ void NDTScanMatcher::callbackSensorPoints(
     ndt_pose_with_covariance_pub_->publish(result_pose_with_cov_msg);
   }
 
-  publishTF(ndt_base_frame_, result_pose_stamped_msg);
+  tf2_broadcaster_.sendTransform(
+    tier4_autoware_utils::pose2transform(result_pose_stamped_msg, ndt_base_frame_));
+}
+
+void NDTScanMatcher::publish_point_cloud(
```

Comment:
It is better to create separate functions for coordinate transformation and publishment.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1757/commits/b0bc5dc94042f41e366686a3dbeb900c11d29d3e